releasenotes/notes/cni-uds-log.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: release-notes/v2
+kind: feature
+area: traffic-management
+issue:
+  - 32437
+
+releaseNotes:
+  - |
+    **Improved** CNI network plugin to send logs to the CNI daemonset. This allows viewing CNI logs using `kubectl logs`, instead of looking at kubelet logs.
```

Comment:
This is surely an improvement.
How will users that used to look at kubelet logs, know to look here?
It is a `change`, not just improvement.

Reply:
Should we log in both places?

Reply:
Istio CNI doc will be revamped, and this will be clearly documented. I think there should be a release notes about CNI beta graduation, as part of headlines for 1.11 release, and this will be mentioned there as well, not just change notes.

Reply:
We log in both places. The UDS code paths sends a copy to CNI daemonset.

Reply:
> We log in both places

The unit test covers verification for this: https://github.com/istio/istio/pull/33926/files#diff-370d69585b4a3d56f5b5e0c3df20124efea862278838276bc5666474a59b4ee6R55